docs/development.md

```diff
@@ -64,58 +61,47 @@ $ pnpm install
 
 ## Building
 
-You can now finally build the Docker images using the following command:
+You can now finally build the Docker images. Navigate to `install/local/dev` and run the following:
 
 ```sh
-$ pnpm dev build
+$ docker-compose -p wikijump -f docker-compose.yaml -f docker-compose.dev.yaml up
 ```
 
-_This might take a surprisingly long time_. Thankfully, Docker's build step is _heavily_ cached. It won't take this long again (unless of course you need to change one of those early steps).
-
-## Development
+_This might take some time_. Thankfully, Docker's build step is _heavily_ cached. 
 
-You can run development mode using the following:
-
-```sh
-$ pnpm dev
-```
-
-This command will run everything in watch mode, and start the Docker containers. However, it will first ask you if you want to build the containers, as in what `pnpm dev build` does. This is asked because building may take a long time.
+> ### For Windows:
+>
+> You may encounter various errors involving file permissions if using Windows-based tools alongside WSL2. It is recommend that you fix these issues by either granting the correct file permissions to any that Windows may have modified, or by re-cloning the repository purely within WSL2.
 
-Once everything has started, you can connect to `http://www.wikijump.localhost/`. Changes you make to the codebase should automatically be applied to the containers, as your machine's filesystem has been "bound" to the containers' filesystem. This is one-way, so a container can't modify your filesystem.
+Once everything has started, you can connect to `http://www.wikijump.localhost/`. Changes you make to the codebase should automatically be applied to the containers, as your machine's filesystem has been "bound" to the containers' filesystem. This is one-way, so a container can't modify your filesystem. Adding new dependencies, however, will require a rebuild.
 
 You can just kill the terminal (`CTRL + C` usually) when you want to stop the server.
 
-If for some reason development resources aren't shutting down correctly, you can run the following:
-
-```sh
-$ pnpm dev clean
-```
-
 If you want to entirely _reset_ the containers, as their data is otherwise persistent even across restarts, you can run the following:
 
 ```sh
-$ pnpm compose down
+$ docker-compose -p wikijump -f docker-compose.yaml -f docker-compose.dev.yaml down
 ```
 
-You can also bring containers up without the JS/TS development mode.
+It's useful to keep track of existing Docker images and containers, and destroy them when you no longer need them, so you don't waste space rebuilding the same image over and over. If you are using Docker Desktop, you can manage containers and images from the GUI. Otherwise, using the command line:
+
 ```
-$ pnpm dev serve
+$ docker container ls  # List containers
+$ docker rm [ID]       # Destroy the container with this ID
+$ docker images        # List images
+$ docker rmi [ID]      # Remove the image with this ID
```

Comment:
New one you could add (just thought of), `docker system prune`, which goes and deletes _everything_ unused. It cleans up a lot of stuff and is normally only used every couple months kind of thing.